src/main/java/optic_fusion1/mcantimalware/check/CheckRegistery.java

```diff
@@ -1,134 +1,81 @@
-package optic_fusion1.mcantimalware.check;

```

Comment:
Idk what intellij did there, I just registered the new CashPloit check.

Reply:
Idk either xD